content/pages/education/eligibility.md

```diff
@@ -79,8 +83,7 @@ Also, if you haven’t used all of your Post-9/11 GI Bill benefits, you may be a
 
 ### Ready to apply?
 
-<div class="wizard-anchor">
-</div>
+<div id="react-applicationStatus"></div>
 
 [Learn about the application process](/education/apply).
```

Comment:
Same as previous comment.

Reply:
@peggygannon this is hidden on the other pages when there is an in progress form, should those be changed to match this, or should this also only be displayed when there is not yet an in progress form?

Reply:
@cehsu Hi, Claire, To confirm, 

--the "Ready to apply?" header and the "Learn about the application process" link should not be displayed on the static pages if an in-progress form message box appears.

Reply:
Thanks for the clarification @peggygannon, @jbalboni has updated this accordingly.